.pipelines/vhd-builder-windows.yaml

```diff
@@ -0,0 +1,24 @@
+trigger: none
+
+jobs:
+- job: build_vhd_windows

```

Comment:
is this pipeline running anywhere so we can see it succeeding before merging the PR?

Reply:
I have this running in my personal VSO instance. I can either try making the project public or finish getting a build set up in the CloudoNativeCompute project.

Reply:
https://msazure.visualstudio.com/CloudNativeCompute/_build?definitionId=90028 is up and running :)